AdobeDocs/commerce-pwa-studio
# Update theme version to 3.4.0

## Description

Updates the `@adobe/gatsby-theme-aio` dependency version to 3.4.0.

This version includes the fix that allows reference-style links in production builds.
These links are used extensively in this project.

## Related Issue

None

## Motivation and Context

Enable reference-style links that contain the baseURL prefix for production builds.

## How Has This Been Tested?

1. Create a local production build: `npm run start:prefix`
2. Navigate to `localhost:9000/commerce/pwa-studio/`
3. Click on a link known to be a reference style link in the underlying markdown source
4. Verify link works

## Screenshots (if appropriate):

## Types of changes




## Checklist:
